goldenmaster/Plugins/TbSimple/Source/TbSimple/Private/Generated/OLink/TbSimpleSimpleArrayInterfaceOLinkClient.cpp

```diff
@@ -316,6 +532,22 @@ void UTbSimpleSimpleArrayInterfaceOLinkClient::applyState(const nlohmann::json&
 			Execute_BroadcastPropIntChanged(this, PropInt);
 		}
 	}
+	if (fields.contains("propInt32"))
```

Comment:
Avoid nested ifs here for brevity and clarity?

if ( fields.contains("propInt32") && PropInt32 != fields["propInt32"].get<TArray<int32>>() )

Reply:
Created issue for this https://github.com/apigear-io/template-unreal/issues/35